ui/admin/app/components/form/worker/create-worker-led/index.hbs

```diff
@@ -170,6 +170,7 @@
       @value={{this.workerConfigText}}
       @hasCopyButton={{true}}
       @hasLineNumbers={{false}}
+      data-test-worker-config
```

Comment:
Note: In order to test this locally, you'll have to update `internal/UI/VERSION` in `boundary` or `boundary-enterprise` in order to get this change